### ISSUE TITLE
Check Content Escaping command fails

### ISSUE THREAD

**Issue by infabo** [sitegeist/fluid-components#132]
```
 ddev typo3cms fluidcomponents:checkContentEscaping
```

![image](https://user-images.githubusercontent.com/3999104/227224647-fac00d11-edca-499f-9b6e-790c4316f1f3.png)

**Comment by s2b** [comment]
This should be fixed in master already. This happens if the command doesn't find any template directories. Could you check dev-master?

**Comment by infabo** [comment]
No, it happens when using typo3-composer-installers 4 or 5.

```
 return strpos($package->getPackagePath(), Environment::getExtensionsPath()) === 0;
```

![image](https://user-images.githubusercontent.com/3999104/227225908-75d99985-499b-4e88-84ba-8cdc98bd621c.png)

**Comment by infabo** [comment]
see https://github.com/sitegeist/fluid-components/blob/3.5.0/Classes/Command/CheckContentEscapingCommand.php#L255

**Comment by infabo** [comment]
see https://github.com/TYPO3/CmsComposerInstallers

**Comment by s2b** [comment]
At least it fails with an error and not silently. I'll look into it, thanks!

**Comment by infabo** [comment]
Works

**Comment by s2b** [comment]
Fixed with 3.5.1. Thanks!
